src/conversion_helper.js

```diff
@@ -522,6 +522,37 @@ function shouldConvertComment(comment, regexArray = globalIgnore, shouldBeUnique
 }
 
 /*
+  Input: Comment
+    "1 3/4 miles"
+  Output: Comment with body processed to fix user formatting
+    "1.75 miles"
+*/
+function preprocessComment(comment) {
+  function fractionProcessor(input) {
+    var mixedRegex = new RegExp((rh.numberRegex + /(?:[\s+-]+)/.source + rh.numberRegex + "/" + rh.numberRegex + /([\s-])/.source), 'gi');
+    var fractionRegex = new RegExp(rh.numberRegex + "/" + rh.numberRegex + /([\s-])/.source, 'gi');
+
+    input = input.replace(mixedRegex, function(p1, p2, p3, p4, p5) {
```

Comment:
I'm a bit confused what p5 represents - would it make sense to change the names of the variables to be more descriptive about what they are? i.e. `p3` could be `numerator` and `p4` could be `denominator`

Reply:
`p1, p2, p3...pN` are the Nth matched groups from the regex, the variables names are by design from [String.prototype.replace()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace)

Reply:
Oh ok, I'm not super familiar with javascript regexes but if that's the convention it's fine :) Thanks for the link